txnsync/txngroups_test.go

```diff
@@ -129,9 +128,11 @@ func TestTxnGroupEncodingSmall(t *testing.T) {
 	require.ElementsMatch(t, inTxnGroups, out)
 }
 
-func TestTxnGroupEncodingLarge(t *testing.T) {
-	dat, err := ioutil.ReadFile("/Users/nicholas/Downloads/txns.txt")
-	require.NoError(t, err)
+func txnGroupsData() ([]transactions.SignedTxGroup, error) {
+	dat, err := ioutil.ReadFile("txns.txt")
```

Comment:
I'm not opposing to create a test-data file that would be used for verification. But if you're creating that, why don't you place it in `./test/testdata/transactions`